src/pages/query.graphql

```diff
@@ -1,412 +1,225 @@
 }
 
-query GetAllDepositsQuery{
-    deposits(first: 1000,orderBy: updateTimestamp,orderDirection: desc) {
+query GetRitualsQueryByStakingProvider($id:Bytes!){
+    rituals(
+        first:1000
+        where: { participants_contains: [$id]}
+    ) {
         id
-        status
-        user{
+        initiator
+        initTimestamp
+        endTimestamp
+        dkgSize
+        dkgStatus
+        authority
+        accessController
+        participants
+        postedAggregations
+        postedTranscripts
+        publicKey
+        threshold
+        transactions(orderBy:timestamp,orderDirection:desc) {
+            description
+            from
             id
-        }
-        amount
-        newDebt
-        actualAmountReceived
-        treasuryFee
-        walletPubKeyHash
-        fundingTxHash
-        fundingOutputIndex
-        blindingFactor
-        refundPubKeyHash
-        refundLocktime
-        vault
-        depositTimestamp
-        updateTimestamp
-        transactions(orderBy: timestamp,orderDirection:desc){
             timestamp
-            txHash
-            from
             to
-            description
+            txHash
         }
     }
-    statsRecord(id:"current"){
-        mintingStatus
-    }
-    tbtctoken(id:"TBTCToken"){
-        totalMint
-        totalBurn
-        totalSupply
-    }
 }
 
-
-query SearchRedemptionQueryById($id: String!){
-    searchRedemption(text:$id) {
-        id,
-        status,
-        user{
+query GetAllRitualsQuery{
+    rituals(first:1000) {
+        id
+        initiator
+        initTimestamp
+        endTimestamp
+        authority
+        accessController
+        dkgSize
+        dkgStatus
+        participants
+        postedAggregations
+        postedTranscripts
+        publicKey
+        threshold
+        transactions(orderBy:timestamp,orderDirection:desc) {
+            description
+            from
             id
-        },
-        amount,
-        walletPubKeyHash,
-        redeemerOutputScript,
-        redemptionTxHash,
-        treasuryFee,
-        txMaxFee,
-        completedTxHash
-        redemptionTimestamp
-        updateTimestamp
-        transactions(orderBy: timestamp,orderDirection:desc){
             timestamp
-            txHash
-            from
             to
-            description
+            txHash
         }
     }
-    tbtctoken(id:"TBTCToken"){
-        totalMint
-        totalBurn
-        totalSupply
+    ritualCounter(id:"Counter") {
+        total
+        unsuccessful
+        successful
+        notEnded
     }
 }
 
-
-query GetRedemptionQueryByUser($user: String, $id: ID,$completedTxHash: Bytes){
-    redemptions (first: 1000,where:{
-        or:[
-            {user : $user},
-            {id: $id},
-            {completedTxHash: $completedTxHash}
-        ]
-    },orderBy: updateTimestamp,orderDirection: desc) {
-        id,
-        status,
-        user{
+query GetAllStakersQuery {
+  appAuthorizations(
+    first: 1000
```

Comment:
Probably this never will be relevant, and definitely not relevant in the current state of the network, but we should keep in mind that graphql queries like this are limited to a max amount of 1000 results.

If the network continues growing, maybe it is worth to take a look to automatic pagination, that is indeed enabled in `.graphclientrc.yml` but not here in the queries.

https://github.com/graphprotocol/graph-client/blob/main/packages/auto-pagination/README.md